### DISCUSSION TITLE
Unable to add some teams as PR reviewer

### DISCUSSION THREAD

**Discussion by baxelrod-bdai** [community/community#57575] [Pull Requests] [answered]
### Select Topic Area

Bug

### Body

When I go to https://github.com/orgs/bdaiinstitute/teams I see all my organization's teams.  (Some of which are administered via Okta, and some via GitHub).  But I am unable to add some of these teams as a reviewer on a PR.  All but 3 just don't show up in the drop down list.  (The teams that do show up are two of the Okta administered teams, and the one GitHub administered team).  Even manually typing in the team name and pressing ENTER in the reviewer field doesn't work.  However, I can "@" mention all of the teams in a comment on a PR. 

Is there some setting or configuration that allows a team to be made a reviewer in a PR?  What is preventing this feature from working?

**Comment by baxelrod-bdai** [comment]
Same issue here: https://github.com/orgs/community/discussions/57241

**Comment by nicholaspshaw** [comment, marked as answer]
@baxelrod-bdai the teams need to have write access on the relevant repositories

**Reply by jacksonbandcamp** [reply]
The teams within my organization have "all-repository write" access under "Organization access" for each repo, but this doesn't seem to allow the teams to be added to review a PR. The teams need to be added under "Direct access" for each repo and be given "read" access as a minimum there. Is there a way to avoid this manual step and give teams the proper permissions across the whole organization?

**Reply by coneybeare** [reply]
I am seeing the same thing. I have all teams setup to all repository write access at the org level, but can't tag teams as reviewers on prs

**Reply by amara-vo** [reply]
> I am seeing the same thing. I have all teams setup to all repository write access at the org level, but can't tag teams as reviewers on prs

I was having the same issue, but then I realized that the GitHub Organization needs to be on a **paid plan** to have access to Team PR reviewers (see: [GitHub Pricing](https://github.com/pricing))

**Reply by eddieyanez** [reply]
> I was having the same issue, but then I realized that the GitHub Organization needs to be on a **paid plan** to have access to Team PR reviewers (see: [GitHub Pricing](https://github.com/pricing))

👆🏻 THIS is the real answer.
